src/views/QuizView/quizView.js

```diff
@@ -111,10 +119,22 @@ function getTime() {
 }
 
 function saveAnswer(answer) {
+  let repeated = false;
   endTime = getTime();
   const relativeTime = endTime - startTime;
   const ans = new Answer(relativeTime, questions[current], answer, false);
-  Answers.push(ans);
+  for (let i = 0; i < Answers.length; i++) {
+    if (Answers[i].Question == ans.Question) {
+      if (answer) {
+        Answers[i].answer = ans.answer;
+      }
+      repeated = true;
+      break;
+    }
+  }
+  if (!repeated) {
+    Answers.push(ans);
+  }
```

Comment:
Better way to implement this part:
Add id property to the Question object (the same as the number of question)
Then, instead of for loop, you can use find method on the answers array to find where ans.Question.id === current 

When the right answer is found, you can simply reassign its answer and isRepeated flag to true